.circleci/config.yml

```diff
@@ -0,0 +1,216 @@
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples STM32F1 Series
+          command: |
+            scons check=examples examples=stm32f1_discovery,nucleo_f103rb,olimexino_stm32
+
+  "ex-stm32f3":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples STM32F3 Series
+          command: |
+            scons check=examples examples=stm32f3_discovery,nucleo_f303k8
+
+  "ex-stm32f4_discovery":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples STM32F4 Discovery Board
+          command: |
+            scons check=examples examples=stm32f4_discovery
+
+  "ex-stm32f4":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples STM32F4 Series (except STM32F4 Discovery Board)
+          command: |
+            scons check=examples examples=stm32f429_discovery,stm32f469_discovery,nucleo_f401re,nucleo_f411re,nucleo_f429zi
+
+  "ex-stm32f7":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples STM32F7 Series
+          command: |
+            scons check=examples examples=stm32f746g_discovery
+
+  "ex-stm32l4":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples STM32L4 Series
+          command: |
+            scons check=examples examples=stm32l476_discovery,nucleo_l476rg
+
+  "ex-avr":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples ATmega Series
+          command: |
+            scons check=examples examples=avr,arduino_uno
+
+  "ex-lpcxpresso":
+    docker:
+      - image: stronglytyped/xpcc-build-arm:latest
+    environment:
+      - SCONS_LIB_DIR: "/usr/local/lib/python2.7/dist-packages/scons-2.5.1"
+      - SCONSFLAGS: "-j3"
+    steps:
+      - checkout
+      - run:
+          name: Examples LPCxpresso Board
+          command: |
+            scons check=examples examples=lpcxpresso
+
+  "ex-linux":
```

Comment:
Why `ex-` and not `example-`? Is it a char limitation somewhere?

Reply:
Yes, in the workflow view of CircleCI
![screen shot 2017-10-12 at 00 51 40](https://user-images.githubusercontent.com/1626990/31471134-8c7d6ea8-aee7-11e7-912c-175a05d90758.png)

Reply:
Would it be acceptable to switch it? E.g. `stm32f4-example`, `atmega-unittest`?